src/store/singing.ts

```diff
@@ -3228,12 +3228,47 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
           const { id, ...noteWithoutId } = note;
           return noteWithoutId;
         });
-      // ノートをJSONにシリアライズしてクリップボードにコピーする
+
+      // 歌詞のみのテキストのコピーとしてノートのテキストを結合
+      const lyricText = selectedNotes.map((note) => note.lyric).join("");
+      // VOICEVOXのノートのペースト用としてノートをJSONにシリアライズ
       const serializedNotes = JSON.stringify(selectedNotes);
-      // クリップボードにテキストとしてコピーする
-      // NOTE: Electronのclipboardも使用する必要ある？
-      await navigator.clipboard.writeText(serializedNotes);
-      logger.info("Copied to clipboard.", serializedNotes);
+
+      // MIMEタイプとしてapplication/jsonとtext/plainを使用してクリップボードにコピーする
+      // web application/vnd.voicevox.song-notes - VOICEVOXでのノート構造を保持してペーストできる
+      // text/plain - 歌詞テキストだけを内部または他のエディタなどで利用できるようにする
+      // web形式からはじまる形式はChromeのみでサポートされている
+      // https://developer.chrome.com/blog/web-custom-formats-for-the-async-clipboard-api?hl=ja
+      try {
+        // ノートコピー＆ペースト用のカスタムMIMEタイプ
+        const customMimeType = "web application/vnd.voicevox.song-notes";
+        const clipboardItems = [
+          new ClipboardItem({
+            // VOICEVOXのノートのペースト用
+            [customMimeType]: new Blob([serializedNotes], {
+              type: customMimeType,
+            }),
+            // 歌詞のみのテキストのコピー用
+            "text/plain": new Blob([lyricText], { type: "text/plain" }),
+          }),
+        ];
+        await navigator.clipboard.write(clipboardItems);
+        logger.info(
+          "Copied to clipboard with custom MIME type.",
+          serializedNotes,
+        );
+      } catch (e) {
+        // カスタムMIMEタイプがサポートされていないブラウザの場合はフォールバック
+        try {
+          await navigator.clipboard.writeText(serializedNotes);
+          logger.info("Fallback to plain text.", e);
+          logger.info("Copied to clipboard as plain text", serializedNotes);
```

Comment:
これ最近気づいたのですが、エラーになる処理の前にロガー書かないと表示されないんですよね 😇 
なのでこうかも？
```suggestion
        try {
          logger.info("Fallback to plain text.", e);
          await navigator.clipboard.writeText(serializedNotes);
          logger.info("Copied to clipboard as plain text", serializedNotes);
```